### PR TITLE
Feature/#38 search bar

### PR DESCRIPTION
**Pull request by razine-bensari**
Search bar with autocomplete implemented. Testing is difficult because the SearchBar component only render the <GooglePlacesAutoComplete/> which is difficult to test. To discuss with team.

Styling has been moved to another file due to the linter.

### CODE REVIEW

File: components/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        onPress={(data, details = null) => {
+          // 'details' is provided when fetchDetails = true
+          this.setMapLocation(details.geometry.location.lat, details.geometry.location.lng);
+        }}
+        getDefaultValue={() => ''}
+        query={{
+          // available options: https://developers.google.com/places/web-service/autocomplete
+          key: apiKey,
+          language: 'en', // language of the results
+          location: '45.496628, -73.578804', // Centered around hall building
```
Comment (Alessandro100) [inline components/SearchBar.tsx]: Get these values from the campus constant

File: components/SearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          radius: '80000',
```
Comment (Alessandro100) [inline components/SearchBar.tsx]: we can probably minimize the radius to make it more relevant for concordia

Comment (razine-bensari) [reply components/SearchBar.tsx]: ill make it a radius of 10km. Is that good? Would you rather have something smaller?

Comment (razine-bensari) [reply components/SearchBar.tsx]: We also nee to consider Loyola campus

**Pull request merged**